UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -32,6 +33,8 @@ public class LengthTests : LengthTestsBase
 
         protected override double FeetInOneMeter => 3.28084;
 
+        protected override double USSurveyFeetInOneMeter => 3.2808333333;
```

Comment:
I think you got this backwards. My sanity check says USS Feet should be about 0.3 meters. Check your conversion funtions. It's very easy to get this backwards.